.github/workflows/update-dependencies.yml

```diff
@@ -13,19 +13,30 @@ jobs:
     - uses: "opensafely-core/setup-action@v1"
       with:
         python-version: "3.8"
-        install-just: true 
-    
+        install-just: true
+
     - uses: actions/create-github-app-token@v1
       id: generate-token
       with:
         app-id: 1031449  # opensafely-core Create PR app
         private-key: ${{ secrets.CREATE_PR_APP_PRIVATE_KEY }}
 
     - uses: bennettoxford/update-dependencies-action@v1
+      id: update
       with:
         token: ${{ steps.generate-token.outputs.token }}
         # note: we don't currently auto update external production dependencies,
         # as vendoring them into job-runner is messy
         update_command: |
           just upgrade dev
           just upgrade-pipeline
+
+    - name: Notify slack of PR
+      if: ${{ steps.update.outputs.pull-request-operation != 'none' }}
+      uses: slackapi/slack-github-action@v2.0.0
```

Comment:
I think our policy is to link to the full sha of an external action? Dependabot will still update it we add a comment:
So: `slackapi/slack-github-action@485a9d42d3a73031f12ec201c457e2162c45d02d  # v2.0.0`

I won't suggest, as then you'll need another review, as this is job-runner.
Have quickly done PR for airlock: https://github.com/opensafely-core/airlock/pull/901

Reply:
And ehrql: https://github.com/opensafely-core/ehrql/pull/2417

Reply:
slackapi is a [Marketplace verified creator](https://github.com/marketplace?query=slackapi&type=actions) so possibly it's also ok with a version?  Our org-level actions permissions include any marketplace verified creator actions as well as github-created ones